src/parser.rs

```diff
@@ -33,38 +33,32 @@ const KEY_ACCEPTABLE_CHARS: &str = "0-9A-Za-z_";
 /// New line chars
 const NEW_LINE_CHARS: &str = "\n\x0c\x0b\x08";
 
-/// Returns a HashMap with special characters to be escaped
-fn escape_sequences<'a>() -> HashMap<&'a str, String> {
-    [
-        ("b", "\x08".to_string()),
-        ("f", "\x0c".to_string()),
-        ("n", "\n".to_string()),
-        ("r", "\r".to_string()),
-        ("t", "\t".to_string()),
-        ("\"", "\"".to_string()),
-        ("\\", "\\".to_string()),
-        ("$", "$".to_string()),
-    ]
-    .iter()
-    .cloned()
-    .collect()
-}
-
-// Sequences that need escaped when dumping string values
-fn sequences_to_scape<'a>() -> HashMap<&'a str, String> {
-    [
-        ("\x08", "\\b".to_string()),
-        ("\x0c", "\\f".to_string()),
-        ("\n", "\\n".to_string()),
-        ("\r", "\\r".to_string()),
-        ("\t", "\\t".to_string()),
-        ("\"", "\\\"".to_string()),
-        ("\\", "\\\\".to_string()),
-        ("$", "\\$".to_string()),
-    ]
-    .iter()
-    .cloned()
-    .collect()
+lazy_static! {
+    /// HashMap with special characters to be escaped
+    static ref ESCAPE_SEQUENCES: HashMap<&'static str, &'static str> = {
```

Comment:
Couldn't we elide the 'static lifetimes given this is static or does lazy static require this?

Reply:
No. It does not compile. `'static` here tells the compiler that the values behind the references, are guaranteed to be part of the data block in the executable